elliottlib/errata_async.py

```diff
@@ -109,7 +109,7 @@ def compute_cve_package_exclusions(cls, attached_builds: Iterable[str], expected
         :return: a dict that key is CVE name, value is another dict with package name as key and 0 as value
         """
         brew_components = {parse_nvr(nvr)["name"] for nvr in attached_builds}
-        missing_brew_brew_components = {c for components in expected_cve_components.values() for c in components} - brew_components
+        missing_brew_brew_components = {c for components in expected_cve_components.values() for c in components if c != 'openshift-golang-builder-container'} - brew_components
```

Comment:
I don't like seeing specific names of things in code like this (though I'm guilty too). How about pulling this out to a constant, where we can supply a helpful comment about why it's there...